app/docs/dev.md

```diff
@@ -1,29 +1,207 @@
+| foot              |   ft   | $0.3048~\mathrm{metre}$                       |
+| yard              |   yd   | $0.9144~\mathrm{metre}$                       |
+| mile              |   mi   | $1609.344~\mathrm{metre}$                     |
+| fluid ounce       |  fl oz | $28.4130625~\mathrm{millilitre}$              |
+| gill              |   gi   | $142.0653125~\mathrm{millilitre}$             |
+| pint              |   pt   | $568.26125~\mathrm{millilitre}$               |
+| quart             |   qt   | $1.1365225~\mathrm{litre}$                    |
+| gallon            |   gal  | $4546.09~\mathrm{litre}$                      |
+| ounce             |   oz   | $28.349523125~\mathrm{gram}$                  |
+| pound             |   lb   | $0.45359237~\mathrm{kilogram}$                |
+| stone             |   st   | $6.35029318~\mathrm{kilogram}$                |
+
+### `strict_syntax`
+
+If `strict_syntax` is set to true then the answer and response must have `*` or `/` between each part of the expressions and exponentiation must be done using `**`, e.g. `a*b/c**2` is accepted but `ab/c^2` is not.
+
+If `strict_syntax` is set to false, then `*` can be omitted and `^` used instead of `**`, e.g. `a*b/c**2` and `ab/c^2` will be considered identical. In this case it is also recommended to list any multicharacter symbols expected to appear in the response as input symbols.
+
+By default `strict_syntax` is set to true.
 
 ## Examples
-*List of example inputs and outputs for this function, each under a different sub-heading*
-
-### Simple Evaluation
-
-```python
-{
-  "example": {
-    "Something": "something"
-  }
-}
-```
-
-```python
-{
-  "example": {
-    "Something": "something"
-  }
-}
-```
\ No newline at end of file
+
+Implemented versions of these examples can be found in the module 'Examples: Evaluation Functions'.
+
+### 1 Checking if a set of quantities match the Buckingham pi theorem
+
+#### a)
+
+In this example the task is: Given $U$, $L$ and $\nu$, suggest a dimensionless group.
+
+For this problem we do not need to predefine any quantities and give exact dimensions. The algorithm assumes that all symbols in the answer (that are not numbers or predefined constants such as $\pi$) are quantities and that there are no other quantities that should appear in the answer.
+
+**Note:** This means that the algorithm does not in any way check that the stated answer is dimensionless, that is left to the task author.
+
+For this example an EXPRESSION response area is used with answer set to `U*L/nu`. It is not necessary to use this specific answer, any example of a correct dimensionless group should work.
+
+With default settings it is required to put `*` (or `/`) between each part of the response and answer. To remove this requirement the grading parameter `strict_syntax` is set to false. Since `nu` is a multicharacter symbol it needs to be added as an input symbol.
```

Comment:
Could we be totally explicit and list the settings together at the end:

answer: `U*L/nu`
parameters: strict_syntax: false

Reply:
Added summarising list of parameters to each example